src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Resources/Base/BuildReleaseArm64XFormsDotNet.apkdesc

```diff
@@ -137,76 +137,85 @@
       "Size": 1774
     },
     "assemblies/UnnamedProject.dll": {
-      "Size": 4990
+      "Size": 5015
     },
     "assemblies/Xamarin.AndroidX.Activity.dll": {
-      "Size": 5944
+      "Size": 13842
     },
     "assemblies/Xamarin.AndroidX.AppCompat.AppCompatResources.dll": {
-      "Size": 6036
+      "Size": 6227
     },
     "assemblies/Xamarin.AndroidX.AppCompat.dll": {
-      "Size": 119847
+      "Size": 134494
     },
     "assemblies/Xamarin.AndroidX.CardView.dll": {
-      "Size": 6802
+      "Size": 6977
     },
     "assemblies/Xamarin.AndroidX.CoordinatorLayout.dll": {
-      "Size": 17260
+      "Size": 17886
     },
     "assemblies/Xamarin.AndroidX.Core.dll": {
-      "Size": 100665
+      "Size": 120018
+    },
+    "assemblies/Xamarin.AndroidX.CursorAdapter.dll": {
+      "Size": 9002
     },
     "assemblies/Xamarin.AndroidX.DrawerLayout.dll": {
-      "Size": 14631
+      "Size": 15330
     },
     "assemblies/Xamarin.AndroidX.Fragment.dll": {
-      "Size": 41736
+      "Size": 45683
     },
     "assemblies/Xamarin.AndroidX.Legacy.Support.Core.UI.dll": {
-      "Size": 6083
+      "Size": 6242
     },
     "assemblies/Xamarin.AndroidX.Lifecycle.Common.dll": {
-      "Size": 6471
+      "Size": 6574
     },
     "assemblies/Xamarin.AndroidX.Lifecycle.LiveData.Core.dll": {
-      "Size": 6618
+      "Size": 6749
     },
     "assemblies/Xamarin.AndroidX.Lifecycle.ViewModel.dll": {
-      "Size": 3071
+      "Size": 6289
     },
     "assemblies/Xamarin.AndroidX.Loader.dll": {
-      "Size": 12926
+      "Size": 13087
+    },
+    "assemblies/Xamarin.AndroidX.MultiDex.dll": {
+      "Size": 8354
     },
     "assemblies/Xamarin.AndroidX.RecyclerView.dll": {
-      "Size": 89997
+      "Size": 93990
     },
     "assemblies/Xamarin.AndroidX.SavedState.dll": {
-      "Size": 4909
+      "Size": 4969
     },
     "assemblies/Xamarin.AndroidX.SwipeRefreshLayout.dll": {
-      "Size": 10575
+      "Size": 13974
     },
     "assemblies/Xamarin.AndroidX.ViewPager.dll": {
-      "Size": 18594
+      "Size": 19073
     },
     "assemblies/Xamarin.Forms.Core.dll": {
-      "Size": 528450
+      "Size": 561410
     },
     "assemblies/Xamarin.Forms.Platform.Android.dll": {
-      "Size": 337925
+      "Size": 372848
     },
     "assemblies/Xamarin.Forms.Platform.dll": {
-      "Size": 11083
+      "Size": 17182
     },
     "assemblies/Xamarin.Forms.Xaml.dll": {
-      "Size": 60774
+      "Size": 63517
     },
     "assemblies/Xamarin.Google.Android.Material.dll": {
-      "Size": 42285
+      "Size": 66417
+    },
+    "assemblies/Xamarin.Google.Guava.ListenableFuture.dll": {
+      "Size": 10640
     },
     "classes.dex": {
-      "Size": 3502580
+      "Size": 5908848
```

Comment:
Xamarin.Forms 5 has a lot newer AndroidX libraries... Which is apparently a lot more Java code.